### PR TITLE
Pass over API App doc [ci skip]

### PR DESCRIPTION
**Pull request by vipulnsward**

### CODE REVIEW

File: guides/source/api_app.md [hunk 1]
```diff
@@ -374,8 +372,7 @@ controller modules by default:
 - `ActionController::Rescue`: Support for `rescue_from`.
 - `ActionController::Instrumentation`: Support for the instrumentation
   hooks defined by Action Controller (see [the instrumentation
-  guide](active_support_instrumentation.html#action-controller) for
-more information regarding this).
+  guide](active_support_instrumentation.html#action-controller) for more information regarding this).
```
Comment (vijaydev) [inline guides/source/api_app.md]: I don't see a change here except the lines merged into one. Was that the intent? No need, imo.

File: guides/source/api_app.md [hunk 2]
```diff
@@ -124,14 +122,14 @@ Of course, the Rails boot process also glues together all registered components.
 For example, the Rails boot process is what uses your `config/database.yml` file
 when configuring Active Record.
 
-**The short version is**: you may not have thought about which parts of Rails
+**The short version is**: you may have not thought about which parts of Rails
```
Comment (vijaydev) [inline guides/source/api_app.md]: the earlier version is correct.

File: guides/source/api_app.md [hunk 3]
```diff
@@ -124,14 +122,14 @@ Of course, the Rails boot process also glues together all registered components.
...
 are still applicable even if you remove the view layer, but the answer turns out
 to be most of it.
 
 The Basic Configuration
 -----------------------
 
-If you're building a Rails application that will be an API server first and
+If you are building a Rails application that will be an API server first and
```
Comment (vijaydev) [inline guides/source/api_app.md]: why? :)

File: guides/source/api_app.md [hunk 4]
```diff
@@ -97,11 +96,10 @@ Handled at the Action Pack layer:
   means not having to spend time thinking about how to model your API in terms
   of HTTP.
 - URL Generation: The flip side of routing is URL generation. A good API based
-  on HTTP includes URLs (see [the GitHub Gist API](http://developer.github.com/v3/gists/)
+  on HTTP, includes URLs (see [the GitHub Gist API](http://developer.github.com/v3/gists/)
```
Comment (vijaydev) [inline guides/source/api_app.md]: shouldn't have a comma here

File: guides/source/api_app.md (hunk 1 above)
Comment (vipulnsward) [inline guides/source/api_app.md]: Yes, wanted to be consistent with formatting.

File: guides/source/api_app.md (hunk 3 above)
Comment (vipulnsward) [inline guides/source/api_app.md]: Just easier to read :-)

File: guides/source/api_app.md (hunk 4 above)
Comment (vipulnsward) [inline guides/source/api_app.md]: Actually, HTTP vs HTTP includes URLs is ambiguous here. I am sure we are not referring to HTTP includes here.

File: guides/source/api_app.md (hunk 2 above)
Comment (vipulnsward) [inline guides/source/api_app.md]: true, fixed

**Comment by kaspth** [comment]
Thanks, but these changes don't really improve the docs. It would be sad to remove the cheekiness here.

File: guides/source/api_app.md (hunk 4 above)
Comment (vijaydev) [inline guides/source/api_app.md]: But a comma isn't the right fit here. I'd not mind a rewrite of that sentence to remove the ambiguity, if any.

**Pull request closed without merging**